ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/router_bgp.py

```diff
@@ -159,6 +164,15 @@ def _peer_groups(self) -> list | None:
 
         return peer_groups
 
+    def _generate_bfd_timers(self, peer_group) -> dict:
+        bfd_timer = {}
+        if self.shared_utils.bgp_peer_groups[peer_group]["bfd"]:
+            # assigning default value here.
+            bfd_timers = {"interval": 2000, "min_rx": 2000, "multiplier": 3}
```

Comment:
Implementing the defaults here means all peer groups will get the same defaults. I think we need to define this in some dict.